addon/templates/components/power-select-infinity.hbs

```diff
@@ -26,7 +26,7 @@
         onchange=onchange
         onclose=onclose
         onfocus=(action "handleFocus")
-        oninput=oninput
+        oninput=(action "onInput")
```

Comment:
@bakerac4  it looks like the consuming app is losing the ability to provide their own `oninput` action handler with this change. maybe we could add a `tryInvoke(this, 'oninput', [text])` to the `onInput` action to maintain this?